sdks/python/apache_beam/typehints/decorators.py

```diff
@@ -743,7 +622,7 @@ def _normalize_var_keyword_hint(hint, arg_name):
     return typehints.Dict[str, typehints.Union[values]]
 
 
-def getcallargs_forhints_impl_py3(func, type_args, type_kwargs):
+def getcallargs_forhints(func, *type_args, **type_kwargs):
```

Comment:
Do we still need asterisks on line 642 given that the signature changed?

Reply:
Yes, asterisks should be needed because signature.bind regard `type_args` as tuple and `type_kwargs` as dict without them.

Reply:
Ah yes, that's right.